script/dump-db

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -euo pipefail
-
 DUMP_FOLDER="/var/lib/openqa/SQL-DUMPS"
```

Comment:
Normally I would suggest to replace `/var/lib` with `${OPENQA_BASEDIR:-/var/lib}`. Maybe we want to keep it explicit here, though (due to the danger of deleting too much files).

Reply:
I would keep it explicit but I also modified the script that the caller can at least adjust these values.